packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFailedPaymentsCancellation/index.stories.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionFailedPaymentsCancellation',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionFailedPaymentsCancellation',
+  'Sent when failed payments result in cancellation of user subscription.',
+  {
+    productName: 'Firefox Fortress',
+    cancellationSurveryUrl:
```

Comment:
This also needs `isCancellationEmail: true`. This will include a link to `Reactivate subscription` in the footer instead of `Cancel subscription`.

Reply:
Good catch!